src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,156 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskException;
+import duke.task.TaskList;
+import duke.task.TaskType;
+import duke.task.TaskTypeDecodeException;
+import duke.task.Todo;
+
+/**
+ * Encapsulates the logic for storing tasks.
+ *
+ * It will create folders and file based on the given filePath in the constructor.
+ *
+ * A connection to the file is created and maintained once this class is instantiated.
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Constructs a Storage class.
+     *
+     * A connection to the file is created and maintained once this class is instantiated.
```

Comment:
Should the empty line between the two sentences be deleted according to the coding standard?

Reply:
Oh I see. Okay, I've changed them as well since the coding standard states it. Thanks!